scripts/SHI/scpSHI_Jetz_complete.yml

```diff
@@ -0,0 +1,93 @@
+script: scpSHI_Jetz_complete.R
+description: 
+  This script loads the data needed to calculate the area and connectivity scores 
```

Comment:
I think straight to: This scripts calculates (a modified?/improved?)the Species Habitat Index (SHI)....
I also think you should add here what is your contribution...